### ISSUE TITLE
Price List widget needs responsive enhancement

### ISSUE THREAD

**Issue by parfilov** [pojome/elementor#4497]
Hello!
There are very poor opportunities to tune responsive layouts of the Price List widget. Especially, it looks ugly when we use an image.
![elementor_price](https://user-images.githubusercontent.com/12493645/40319592-800acc30-5d4a-11e8-8fed-25dd3d0bca1f.png)

Do something with this please.

**Comment by 319heads** [comment]
Yes, Price List widget is still currently unusable with images due to no mobile support. Would love to see this widget get some attention.
